conbench/api/runs.py

```diff
@@ -205,17 +207,26 @@ def get(self):
             name: sha
             schema:
               type: string
+          - in: query
+            name: days
+            schema:
+              type: integer
         tags:
           - Runs
         """
         sha_arg: Optional[str] = f.request.args.get("sha")
         commit_hashes = sha_arg.split(",") if sha_arg else None
 
+        days_arg: Optional[int] = f.request.args.get("days")
+        days = int(days_arg) if days_arg else 14
+        if days > 30:
+            self.abort_400_bad_request("days must be no more than 30")
```

Comment:
Long-term we definitely want a way to get older data, but an offset to provide pagination here is probably the solution we want